be/src/exec/pipeline/aggregate/aggregate_blocking_source_operator.cpp

```diff
@@ -0,0 +1,64 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "aggregate_blocking_source_operator.h"
+
+namespace starrocks::pipeline {
+
+bool AggregateBlockingSourceOperator::has_output() const {
```

Comment:
is_ht_done returning false means hash table's building is unfinished, so chunk is cannot pulled out.

Reply:
adjust is_ht_done to is_ht_eos。true means all data in the hash table has been taken out